### ISSUE TITLE
Meeting Notes

### ISSUE THREAD

**Issue by ramess101** [ramess101/RJMC_LJ_Ethane#2]
@ocmadin 

5/17 Notes:

- Owen is going to review the RJMC_LJ_2CLJ.py code to understand the details of what is going on and to determine ways to improve the code.

- We concluded that we will use the transition matrix with a delayed acceptance RJMC approach. We will run this by Josh and John on Wednesday.

- Transition matrix will require a way to scale epsilon, sigma, for different bond lengths and also different quadrupoles

- If we do not use a transition matrix but just use a delayed acceptance, we were thinking that we would want the step size to revert to the original guess so that it can find the optimal in 100 MCMC steps, or so. We would store the tuned step size for after the delayed acceptance criteria is applied, so that we do not need a completely new burn-in period. However, we foresee some problems if we have a non-informative prior that it will take a while to find the optimal region. 

- By using a transition matrix and a delayed acceptance, we should not need to use a different step size for the model swap stage

Tasks:

- [ ] Modify RJMC_tuned to allow for delayed acceptance

- [ ] Modify posterior so that accounts for different number of parameters

- [ ] Modify AUA so that bond length is actually a parameter

- [ ] Modify LJ so that it is actually 2LJ (i.e. eps = 2*eps, bond length=0). This would only really help if we do not use a transition matrix

- [ ] Use sigma+L/2 as the property that is scaled. This would also be better if no transition matrix so that sigma is more feasible.

**Comment by ramess101** [comment]
@ocmadin 

Based on the discussion yesterday with @maxentile , we should try to implement Annealing Importance Sampling. Otherwise, we would need to ensure that we maintain detailed balance with the delayed acceptance approach, which is likely not worth it since Josh is working on AIS anyways.

Also, John Codera suggested that to check that our Jacobian is correct we can generate data from each model in a given ratio and see if RJMC recovers that ratio.

**Comment by ocmadin** [comment]
I agree.  I will read up on AIS and work towards getting it up and running.
